source/class/qx/tool/cli/api/CompilerApi.js

```diff
@@ -56,8 +56,8 @@ qx.Class.define("qx.tool.cli.api.CompilerApi", {
     
     /**
      * Register compiler tests
-     * @param  cmd: {qx.tool.cli.commands.Command} current command
-     * @return {Promise<void>}
+     * @param {qx.tool.cli.commands.Command} cmd  current command
+     * @async
```

Comment:
why are we adding `@async` in the jsdoc?  If this is needed for API viewer etc, it should be output as the meta data by the compiler, because the jsdoc will always be out of date and unreliable.

Reply:
I agree, jsdoc should add what cannot be taken from the code itself. Async functions are defined by returning a Promise, so the attention should be on annotating the return value. It would be great if we could update the API viewer to document the type of the value returned by the Promise one day, because that's what really matters for the API consumer.